ui/app/AppLayouts/Profile/stores/RootStore.qml

```diff
@@ -247,7 +247,7 @@ QtObject {
     }
 
     function getEnsRegistry() {
-        return profileModelInst.ens.getENSRegistry()
+        return "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e"
```

Comment:
I guess we can put it in Constants at that point

Reply:
Fair, I've been lazy ;)